src/module/actor/lancer-actor.ts

```diff
@@ -249,6 +249,8 @@ export class LancerActor extends Actor {
       this.system.grit = Math.ceil(this.system.level / 2);
       this.system.hp.max = 6 + this.system.grit;
```

Comment:
So unrelated to the PR, but @Eranziel did you know that there’s a [rules.json](https://github.com/massif-press/lancer-data/blob/master/lib/rules.json#L6) file in lancer-data that has some of these baselines already defined? Probably unlikely to change in the future but it might be helpful to avoid “magic numbers” in hardcoding?

Reply:
I think I did know about that, but had forgotten about it! Thanks for the reminder. I'll see about integrating those where we can.